.github/workflows/release.yml

```diff
@@ -9,28 +9,23 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: msys2/setup-msys2@v2
-    - name: Install dependencies
-      run: |
-         msys2 -c 'pacman -S --noconfirm libxslt base-devel'
     - name: Add msbuild to PATH
       uses: microsoft/setup-msbuild@v1.0.2
+    - name: Install msys2 dependencies
+      run: |
+        c:\msys64\msys2_shell.cmd -defterm -no-start -here -mingw64 -c 'pacman -S --noconfirm libxslt'
```

Comment:
I don't have a run of this on the SL repo to check but is it still downloading and reinstalling heaps of dependencies?

Reply:
I don't think that will.

Reply:
Ran it locally, seems to just update libxslt.